### PR TITLE
Added link to Db Module

### PR DESCRIPTION
**Pull request by ThomasLandauer**

### CODE REVIEW

File: ext/Recorder.php [hunk 1]
```diff
@@ -16,8 +16,6 @@
  * Saves a screenshot of each step in acceptance tests and shows them as a slideshow on one HTML page.
  * Activated only for suites with WebDriver module enabled.
  *
- *  ![recorder](http://codeception.com/images/recorder.gif)
```
Comment (DavertMik) [inline ext/Recorder.php]: why do you remove the picture? Its nice ;)

Comment (ThomasLandauer) [inline ext/Recorder.php]: No, it's really annoying! Even worse than most ads ;-) And it doesn't add any value. Everybody knows what a screenshot looks like...

Comment (DavertMik) [inline ext/Recorder.php]: Then please make it as a link.
Recorder is much easier to understand when you see it in action.

Comment (ThomasLandauer) [inline ext/Recorder.php]: OK, good compromise :-) Did that.

**Comment by DavertMik** [comment]
Thanks

**Pull request merged**